packages/nimble-components/src/combobox/specs/IxD.md

```diff
@@ -0,0 +1,238 @@
+# Nimble Combobox (IxD)
+
+## Overview
+
+The combobox component is used to present a pre-defined list of options that end-users can choose from or type to create a custom value. Comboboxes combine the flexibility of text input with the efficiency of selection for more user-friendly interactions.
+
+> THE VISUAL DESIGN IN THIS DOCUMENT MAY NOT BE ACCURATE
+
+### Background
+
+-   [Select and Combobox Refresh HLD (#1303)](https://github.com/ni/nimble/issues/1303)
+-   [ViD Combobox Styles (Figma)](https://www.figma.com/file/PO9mFOu5BCl8aJvFchEeuN/Nimble_Components?type=design&node-id=1529-41777&mode=design)
+
+## Usage
+
+When to use:
+
+-   To provide an input for text entry coupled with a popup for quick selection and filtering
+-   To allow end-users to enter custom values
+-   For entering settings or preference values that don't (yet) exist
+-   For single selection
+
+When not to use:
+
+-   When text entry is not needed for selection
+-   When there are fewer than three options
+-   For complex and/or hierarchical data
+-   For multi selection
+
+### Anatomy
+
+![ ](spec-images/combobox-1.png)
+
+| Element     | Description                                                             |
+| ----------- | ----------------------------------------------------------------------- |
+| Label       | Text that tells the end-user what to expect in the component            |
+| Field input | Input that displays the current selected or entered option              |
+| Menu button | Click area that opens the menu                                          |
+| Menu        | A list of options to choose from displayed as an _open_ state           |
+| Menu item   | A selection the end-user can make, shown with other options in the menu |
+
+#### Label
+
+Comboboxes should always have a label, unless approved by an interaction designer or user researcher (usually in toolbar use cases). _See [Select](/packages/nimble-components/src/select/specs/IxD.md) for more information._
+
+#### Field Input
+
+Comboboxes can be empty by default or have a default selection from the menu. Empty comboboxes should include placeholder text that propts the end-user to enter a custom value or select an option from the menu. Placeholder text should always follow the pattern "Enter or select [thing(s)]", for example "Enter or select address". Ellipses are not needed. Use sentence casing.
```

Comment:
>Ellipses are not needed. Use sentence casing.

Could you clarify whether these are guidelines for the placeholder or the value (or both)?

Reply:
Clarified in file.